src/core/CoolBoardSensors.cpp

```diff
@@ -72,31 +72,38 @@ void CoolBoardSensors::read(JsonObject &root) {
   delay(100);
 
   if (this->lightDataActive.visible) {
```

Comment:
we should just switch to one boolean here indicating wether the SI114X is active or not instead of 3 booleans

Reply:
Only for SI114X ?

Reply:
Everything you can think of (in my opinion)